src/components/data-pane/data-selector.tsx

```diff
@@ -48,8 +48,9 @@ export class DataSelectorBase extends React.PureComponent<DataSelectorProps, any
         <Modal
          isOpen={this.state.modalIsOpen}
          onRequestClose={this.closeModal}
-         contentLabel="Data Selector"
-         styleName="modal"
+         contentLabel='Data Selector'
```

Comment:
@felixcodes Maybe it's worth researching appropriate TSLint flag (in another PR) to enforce consistency for this.